src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -2749,7 +2758,17 @@ def create_image_version(cmd, resource_group_name, gallery_name, gallery_image_n
                                             target_regions=target_regions or [TargetRegion(name=location)],
                                             source=source, replica_count=replica_count,
                                             storage_account_type=storage_account_type)
-    image_version = ImageVersion(publishing_profile=profile, location=location, tags=(tags or {}))
+    if cmd.supported_api_version(min_api='2019-07-01', operation_group='gallery_image_versions'):
+        GalleryImageVersionStorageProfile = cmd.get_models('GalleryImageVersionStorageProfile')
+        GalleryArtifactVersionSource = cmd.get_models('GalleryArtifactVersionSource')
+        # GalleryOSDiskImage = cmd.get_models('GalleryOSDiskImage')
```

Comment:
why do we need these two lines?

Reply:
I will add new type of source.

Reply:
do you mean in the future?

Reply:
I will delete them and keep them in my private notebook.